### PR TITLE
[docs] Batch16 - Quantiles

### PR DESCRIPTION
**Pull request by edgar2020**
### Description

The PR adds examples for functions `DS_GET_QUANTILE`, `DS_GET_QUANTILES`, `DS_HISTOGRAM`, `DS_CDF`, `DS_RANK`, `APPROX_QUANTILE`, `APPROX_QUANTILE_DS`, `APPROX_QUANTILE_FIXED_BUCKETS` , `DS_QUANTILES_SKETCH`, and `DS_QUANTILE_SUMMARY`.

This PR has:

- [X] been self-reviewed.
- [X] Link-linter passed
- [X] Spellchecker passed

### CODE REVIEW

File: docs/querying/sql-functions.md [hunk 1]
```diff
@@ -1027,36 +1068,108 @@ Returns the result of integer division of `x` by `y`.
...
+FROM "flight-carriers"
+
+```
+
+May return the following:
 
-Returns a string representing an approximation to the histogram given the specified bin definition.
+| `estimate_histogram` | 
+| -- |
+| `[354396.0,150199.00000000003,39848.000000000015,21694.99999999997]` |
```
Comment (ektravel) [inline docs/querying/sql-functions.md]:
```suggestion
| `[358496.0,153974.99999999997,39909.99999999999,13757.000000000005]` |
```

File: docs/querying/sql-functions.md [hunk 2]
```diff
@@ -1068,27 +1181,109 @@ Creates an HLL sketch on a column containing HLL sketches or a regular column.
...
+```sql
+SELECT DS_RANK( DS_QUANTILES_SKETCH("Distance"), 500) AS "estimate_rank"
+FROM "flight-carriers"
+```
+
+May return the following:
+
+| `estimate_rank` |
+| -- |
+| `0.43791089804959216` |
```
Comment (ektravel) [inline docs/querying/sql-functions.md]:
```suggestion
| `0.43837721544923675 ` |
```

File: docs/querying/sql-functions.md [hunk 3]
```diff
@@ -133,27 +133,68 @@ Counts distinct values of a Theta sketch column or a regular column.
 
 ## APPROX_QUANTILE
 
-`APPROX_QUANTILE(expr, <NUMERIC>, [<NUMERIC>])`
+:::info
+Deprecated in favor of [`APPROX_QUANTILE_DS`](#approx_quantile_ds).
+:::
 
-**Function type:** [Aggregation](sql-aggregations.md)
+* **Syntax:** `APPROX_QUANTILE(expr, fraction, [k])`
```
Comment (ektravel) [inline docs/querying/sql-functions.md]:
```suggestion
* **Syntax:** `APPROX_QUANTILE(expr, probability, [k])`
```
Changed `fraction` to `probability` to match what's in [Quantiles sketch functions](https://druid.apache.org/docs/latest/querying/sql-aggregations/#quantiles-sketch-functions).

File: docs/querying/sql-functions.md [hunk 4]
```diff
@@ -133,27 +133,68 @@ Counts distinct values of a Theta sketch column or a regular column.
...
-Deprecated in favor of `APPROX_QUANTILE_DS`.
+[Learn more](sql-aggregations.md)
 
 ## APPROX_QUANTILE_DS
 
-`APPROX_QUANTILE_DS(expr, <NUMERIC>, [<NUMERIC>])`
+Computes approximate quantiles on a Quantiles sketch column or a regular numeric column. The parameter `k` determines the accuracy and size of the sketch. The [DataSketches Quantiles Sketch module](../development/extensions-core/datasketches-quantiles.md) provides more information on the parameter `k`.
 
-**Function type:** [Aggregation](sql-aggregations.md)
+* **Syntax:** `APPROX_QUANTILE_DS(expr, fraction, [k])`
```
Comment (ektravel) [inline docs/querying/sql-functions.md]:
```suggestion
* **Syntax:** `APPROX_QUANTILE_DS(expr, probability, [k])`
```
Changed `fraction` to `probability` to match what's in [Quantiles sketch functions](https://druid.apache.org/docs/latest/querying/sql-aggregations/#quantiles-sketch-functions).

File: docs/querying/sql-functions.md [hunk 5]
```diff
@@ -133,27 +133,68 @@ Counts distinct values of a Theta sketch column or a regular column.
...
+* **Syntax:** `APPROX_QUANTILE(expr, fraction, [k])`
+* **Function type:** Aggregation
 
-Deprecated in favor of `APPROX_QUANTILE_DS`.
+[Learn more](sql-aggregations.md)
 
 ## APPROX_QUANTILE_DS
 
-`APPROX_QUANTILE_DS(expr, <NUMERIC>, [<NUMERIC>])`
+Computes approximate quantiles on a Quantiles sketch column or a regular numeric column. The parameter `k` determines the accuracy and size of the sketch. The [DataSketches Quantiles Sketch module](../development/extensions-core/datasketches-quantiles.md) provides more information on the parameter `k`.
```
Comment (ektravel) [inline docs/querying/sql-functions.md]:
```suggestion
Computes approximate quantiles on a Quantiles sketch column or a regular numeric column. See [DataSketches Quantiles Sketch module](../development/extensions-core/datasketches-quantiles.md) for a description of parameters.
```

File: docs/querying/sql-functions.md [hunk 6]
```diff
@@ -133,27 +133,68 @@ Counts distinct values of a Theta sketch column or a regular column.
...
 
-**Function type:** [Aggregation](sql-aggregations.md)
+* **Syntax:** `APPROX_QUANTILE_DS(expr, fraction, [k])`
+* **Function type:** Aggregation
+
+
+<details><summary>Example</summary>
+
+The following example approximates the median of the `Distance` column from the `flight-carriers` datasource using a `k` value of 128. The query may return a different approximation on each execution.
```
Comment (ektravel) [inline docs/querying/sql-functions.md]:
```suggestion
The following example approximates the median of the `Distance` column from the `flight-carriers` datasource. The query may return a different approximation on each execution.
```

File: docs/querying/sql-functions.md [hunk 7]
```diff
@@ -133,27 +133,68 @@ Counts distinct values of a Theta sketch column or a regular column.
...
+
+</details>
 
-Computes approximate quantiles on a Quantiles sketch column or a regular numeric column.
+[Learn more](sql-aggregations.md)
 
 ## APPROX_QUANTILE_FIXED_BUCKETS
 
-`APPROX_QUANTILE_FIXED_BUCKETS(expr, <NUMERIC>, <NUMERIC>, <NUMERIC>, <NUMERIC>, [<CHARACTER>])`
+Computes approximate quantiles on fixed buckets histogram column or a regular numeric column. The [fixed buckets histogram](../development/extensions-core/approximate-histograms.md#fixed-buckets-histogram) documentation describes the parameters.
```
Comment (ektravel) [inline docs/querying/sql-functions.md]:
```suggestion
Computes approximate quantiles on fixed buckets histogram column or a regular numeric column. See [Fixed buckets histogram](../development/extensions-core/approximate-histograms.md#fixed-buckets-histogram) for a description of parameters.
```

File: docs/querying/sql-functions.md [hunk 8]
```diff
@@ -133,27 +133,68 @@ Counts distinct values of a Theta sketch column or a regular column.
...
 
-**Function type:** [Aggregation](sql-aggregations.md)
+* **Syntax:** `APPROX_QUANTILE_FIXED_BUCKETS(expr, probability, numBuckets, lowerLimit, upperLimit, [outlierHandlingMode])`
+* **Function type:** Aggregation
+
+
+<details><summary>Example</summary>
+
+The following example approximates the median of a histogram on the `Distance` column from the `flight-carriers` datasource. The histogram has 10 buckets, a lower limit of zero, an upper limit of 2500, and outliers are ignored.
```
Comment (ektravel) [inline docs/querying/sql-functions.md]:
```suggestion
The following example approximates the median of a histogram on the `Distance` column from the `flight-carriers` datasource. The histogram has 10 buckets, a lower limit of zero, an upper limit of 2500, and ignores outlier values. 
```

File: docs/querying/sql-functions.md [hunk 9]
```diff
@@ -1027,36 +1068,108 @@ Returns the result of integer division of `x` by `y`.
...
+| `[0.6332237016416492,0.8908411023460711,0.9612303007393957,1.0]` |
+
+</details>
+
+[Learn more](sql-scalar.md#sketch-functions)
 
 ## DS_GET_QUANTILE
 
-`DS_GET_QUANTILE(expr, fraction)`
+Returns the quantile estimate corresponding to `fraction` from a Quantiles sketch.
```
Comment (ektravel) [inline docs/querying/sql-functions.md]:
```suggestion
Returns the quantile estimate corresponding to the fraction from a Quantiles sketch. 
```

File: docs/querying/sql-functions.md [hunk 10]
```diff
@@ -1027,36 +1068,108 @@ Returns the result of integer division of `x` by `y`.
...
 
-Returns a string representing an array of quantile estimates corresponding to a list of fractions from a quantiles sketch.
+</details>
+
+[Learn more](sql-scalar.md#sketch-functions)
 
 ## DS_HISTOGRAM
 
-`DS_HISTOGRAM(expr, splitPoint0, splitPoint1, ...)`
+Returns a string representing an approximation to the histogram given a list of split points that define the histogram bins from a Quantiles sketch.
```
Comment (ektravel) [inline docs/querying/sql-functions.md]:
```suggestion
Returns an approximation to the histogram from a Quantiles sketch. The split points define the histogram bins. 
```

File: docs/querying/sql-functions.md [hunk 11]
```diff
@@ -1068,27 +1181,109 @@ Creates an HLL sketch on a column containing HLL sketches or a regular column.
...
 
-`DS_QUANTILE_SUMMARY(expr)`
+Returns a string summary of a Quantiles sketch. 
+* **Syntax:** `DS_QUANTILE_SUMMARY(expr)`
+* **Function type:** Scalar, sketch
 
-**Function type:** [Scalar, sketch](sql-scalar.md#sketch-functions)
+<details><summary>Example</summary>
+
+The following example returns a summary of a Quantiles sketch on the `Distance` column from the `flight-carriers`.
```
Comment (ektravel) [inline docs/querying/sql-functions.md]:
```suggestion
The following example returns a summary of a Quantiles sketch on the `Distance` column from the `flight-carriers` datasource.
```

File: docs/querying/sql-functions.md [hunk 12]
```diff
@@ -1068,27 +1181,109 @@ Creates an HLL sketch on a column containing HLL sketches or a regular column.
...
+</table>
+
+</details>
+
+[Learn more](sql-scalar.md#sketch-functions)
 
 ## DS_QUANTILES_SKETCH
 
-`DS_QUANTILES_SKETCH(expr, [k])`
+Creates a Quantiles sketch on a column containing Quantiles sketches or a regular column. The parameter `k` determines the accuracy and size of the sketch. The [DataSketches Quantiles Sketch module](../development/extensions-core/datasketches-quantiles.md) provides more information on the parameter `k`.
```
Comment (ektravel) [inline docs/querying/sql-functions.md]:
```suggestion
Creates a Quantiles sketch on a Quantiles sketch column or a regular column. See [DataSketches Quantiles Sketch module](../development/extensions-core/datasketches-quantiles.md) for a description of parameters.
```

File: docs/querying/sql-functions.md [hunk 13]
```diff
@@ -1068,27 +1181,109 @@ Creates an HLL sketch on a column containing HLL sketches or a regular column.
...
 
-Creates a Quantiles sketch on a column containing Quantiles sketches or a regular column.
+</details>
+
+[Learn more](sql-aggregations.md)
 
 ## DS_RANK
 
-`DS_RANK(expr, value)`
+Returns an approximate rank for `value` between zero and one, in which the rank signifies the fraction of the distribution less than `value`.
```
Comment (ektravel) [inline docs/querying/sql-functions.md]:
```suggestion
Returns an approximate rank of a given value in a distribution. The rank represents the fraction of the distribution less than the given value.
```

File: docs/querying/sql-functions.md [hunk 14]
```diff
@@ -1068,27 +1181,109 @@ Creates an HLL sketch on a column containing HLL sketches or a regular column.
...
 
-**Function type:** [Scalar, sketch](sql-scalar.md#sketch-functions)
+* **Syntax:** `DS_RANK(expr, value)`
+* **Function type:** Scalar, sketch
+
+<details><summary>Example</summary>
+
+The following example approximates what fraction of records have a lesser value than `500` in the `Distance` column from the `flight-carries` datasource. The query may return a different approximation on each execution.
```
Comment (ektravel) [inline docs/querying/sql-functions.md]:
```suggestion
The following example estimates the fraction of records in the `flight-carriers` datasource where the value in the `Distance` column is less than 500. The query may return a different approximation on each execution.
```

File: docs/querying/sql-functions.md [hunk 15]
```diff
@@ -1122,7 +1317,7 @@ Returns an intersection of Tuple sketches which each contain an array of double
 
 **Function type:** [Scalar, sketch](sql-scalar.md#tuple-sketch-functions)
 
-Computes approximate sums of the values contained within a Tuple sketch which contains an array of double values as the Summary Object.
+Computes approximate sums of the values contained within a Tuple sketch which contains an array of double values as the Summary Object.
```
Comment (ektravel) [inline docs/querying/sql-functions.md]:
```suggestion
Computes approximate sums of the values contained within a Tuple sketch which contains an array of double values as the Summary Object.
```

**Review by ektravel — COMMENTED** [review]
Edited text to improve readability. Tested all of the code examples.

File: docs/querying/sql-functions.md [hunk 16]
```diff
@@ -133,27 +133,68 @@ Counts distinct values of a Theta sketch column or a regular column.
...
+<details><summary>Example</summary>
+
+The following example approximates the median of the `Distance` column from the `flight-carriers` datasource using a `k` value of 128. The query may return a different approximation on each execution.
+
+```sql
+SELECT APPROX_QUANTILE_DS("Distance", 0.5, 128)  AS "estimate_median"
+FROM "flight-carriers"
+```
+
+May return the following:
```
Comment (techdocsmith) [inline docs/querying/sql-functions.md]:
```suggestion
Returns a result similar to the following:
```

File: docs/querying/sql-functions.md [hunk 17]
```diff
@@ -1027,36 +1068,108 @@ Returns the result of integer division of `x` by `y`.
...
+<details><summary>Example</summary>
+
+The following example specifies three split points to return cumulative distribution function approximations on the `Distance` column from the `flight-carriers` datasource. The query may return a different approximation for each bin on each execution.
+
+```sql 
+SELECT DS_CDF( DS_QUANTILES_SKETCH("Distance"), 750, 1500, 2250) AS "estimate_cdf"
+FROM "flight-carriers"
+```
+
+May return the following:
```
Comment (techdocsmith) [inline docs/querying/sql-functions.md]:
```suggestion
Returns a result similar to the following:

File: docs/querying/sql-functions.md [hunk 18]
```diff
@@ -1068,27 +1181,109 @@ Creates an HLL sketch on a column containing HLL sketches or a regular column.
...
+<details><summary>Example</summary>
+
+The following example approximates what fraction of records have a lesser value than `500` in the `Distance` column from the `flight-carries` datasource. The query may return a different approximation on each execution.
+
+```sql
+SELECT DS_RANK( DS_QUANTILES_SKETCH("Distance"), 500) AS "estimate_rank"
+FROM "flight-carriers"
+```
+
+May return the following:
```
Comment (techdocsmith) [inline docs/querying/sql-functions.md]:
```suggestion
Returns a result similar to the following:
```

File: docs/querying/sql-functions.md [hunk 19]
```diff
@@ -1027,36 +1068,108 @@ Returns the result of integer division of `x` by `y`.
...
 
-Returns the quantile estimate corresponding to `fraction` from a quantiles sketch.
+The following example approximates the median of the `Distance` column from the `flight-carriers` datasource. The query may return a different approximation with each execution.
+
+```sql
+SELECT DS_GET_QUANTILE( DS_QUANTILES_SKETCH("Distance"), 0.5) AS "estimate_median"
+FROM "flight-carriers"
+```
+
+May return the following:
```
Comment (techdocsmith) [inline docs/querying/sql-functions.md]:
```suggestion
Returns a result similar to the following:
```

File: docs/querying/sql-functions.md [hunk 20]
```diff
@@ -1027,36 +1068,108 @@ Returns the result of integer division of `x` by `y`.
...
+<details><summary>Example</summary>
+
+The following example approximates the 25th, 50th, and 75th percentiles of the `Distance` column from the `flight-carriers` datasource. The query may return a different approximation for each percentile on each execution.
+
+```sql 
+SELECT DS_GET_QUANTILES( DS_QUANTILES_SKETCH("Distance"), 0.25, 0.5, 0.75) AS "estimate_fractions"
+FROM "flight-carriers"
+```
+
+May returns the following:
```
Comment (techdocsmith) [inline docs/querying/sql-functions.md]:
```suggestion
Returns a result similar to the following:
```

File: docs/querying/sql-functions.md [hunk 21]
```diff
@@ -1027,36 +1068,108 @@ Returns the result of integer division of `x` by `y`.
...
+
+The following example specifies three split points to approximate a histogram on the `Distance` column from the `flight-carriers` datasource. The query may return a different approximation for each bin on each execution.
+
+```sql
+SELECT DS_HISTOGRAM( DS_QUANTILES_SKETCH("Distance"), 750, 1500, 2250) AS "estimate_histogram"
+FROM "flight-carriers"
+
+```
+
+May return the following:
```
Comment (techdocsmith) [inline docs/querying/sql-functions.md]:
```suggestion
Returns a result similar to the following:
```

**Review by techdocsmith — APPROVED** [review]
LGTM

**Pull request merged**